lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/openturns/ProjectionStrategyImplementation.hxx

```diff
@@ -151,6 +165,12 @@ protected:
 
   // Proxy to speed up evaluations of the basis over the input sample
   DesignProxy proxy_;
+
+  // isLeastSquares_ is true if the projection is a regression
```

Comment:
Are these attributes mandatory if their value can be hardcoded into the corresponding property accessors?

Reply:
I removed the attributes in 8d8ebe32519c39c30f3989424581f6336673145e. For example, the [ProjectionStrategyImplementation.cxx#L291](https://github.com/mbaudin47/openturns/blob/d2d18e244254b79129c880b1857a7c416ff819f3/lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/ProjectionStrategyImplementation.cxx#L291) throws and exception. Then [LeastSquaresStrategy.cxx#L176](https://github.com/mbaudin47/openturns/blob/d2d18e244254b79129c880b1857a7c416ff819f3/lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/LeastSquaresStrategy.cxx#L176) overrides this method and returns a hard-coded boolean.